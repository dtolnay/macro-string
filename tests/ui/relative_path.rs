use macro_string_eval::eval;

const _: &str = eval!(include_str!("relative/path.rs"));

fn main() {}
