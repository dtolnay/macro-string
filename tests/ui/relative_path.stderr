error: a relative path is not supported here; use `include_str!(concat!(env!("CARGO_MANIFEST_DIR"), ...))`
 --> tests/ui/relative_path.rs:3:23
  |
3 | const _: &str = eval!(include_str!("relative/path.rs"));
  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
